rmc/utils/mpc.py

```diff
@@ -702,18 +700,25 @@ def calculate_fitted_scores(
     variables = variable_dict.keys()
     interactions_dict = {k: v for k, v in mpc_rel_vars.items() if interaction_char in k}
 
+    context_ht_annots = {"oe", "polyphen"}.intersection(variables)
```

Comment:
I like adding a check here, but this doesn't check that the values exist on the context HT, right?

Maybe this check could get combined with the one below to look something like: 
```
context_ht_annots = {"oe", "polyphen"}.intersection(variables).union(helper_annots)
context_ht = context_with_oe.versions[freeze].ht()
annots_check = {field for field in context_ht.row} - context_ht_annots
if len(annots_check) != 0:
    raise DataException(f"{annots_check} missing from context HT!")
```
I also wonder if we should pass in some of these hardcoded sets (e.g., `context_ht_annots`, `helper_annots`) as args with default values?